src/Files.App/UserControls/SideBar/SideBarControls.xaml

```diff
@@ -85,13 +86,16 @@
 							ToolTipService.ToolTip="{Binding ToolTip, Mode=OneWay}"
 							UseSystemFocusVisuals="True">
 							<Grid.ColumnDefinitions>
-								<ColumnDefinition x:Name="IconColumn" Width="40" />
```

Comment:
Is there no value in keeping the name for the IconColumn for reading?

Reply:
Element names that help identify with what we see visually is useful, not only when you need to refer to the element in code.

Reply:
Having Name creates a field internally in generated partial class. So I'd like to avoid making unnecessary Name.

Reply:
> Having Name creates a field internally in generated partial class. So I'd like to avoid making unnecessary Name.

Comments are helpful too

Reply:
Yeah, that's true.